gammapy/maps/tests/test_wcs.py

```diff
@@ -332,3 +333,15 @@ def test_wcs_geom_equal(npix, binsz, coordsys, proj, skypos, axes, result):
 
     assert (geom0 == geom1) is result
     assert (geom0 != geom1) is not result
+
+@pytest.mark.parametrize("node_type", ["edges", "center"])
+@pytest.mark.parametrize("interp", ["log", "lin", "sqrt"])
+def test_read_write(node_type, interp):
+    energy_axis = MapAxis(nodes=np.logspace(-2., 2, 5), unit='TeV', name='energy', node_type=node_type, interp=interp)
+    time_axis = MapAxis(nodes=np.linspace(0.0, 1000.0, 4), unit='s', name='time', node_type=node_type, interp=interp)
+    geom = WcsGeom.create(skydir=(0, 0), binsz=1, npix=10, axes=[energy_axis, time_axis])
+    m = Map.from_geom(geom, unit='m2')
+    m.write('test.fits', overwrite=True)
```

Comment:
Added assert for the INTERP keywords.
I have still kept the fits r/w (now in tmpdir) because it is a few lines of code to check the `geom` otherwise.